fml/message_loop_task_queues.cc

```diff
@@ -246,7 +246,7 @@ bool MessageLoopTaskQueues::Unmerge(TaskQueueId owner) {
 bool MessageLoopTaskQueues::Owns(TaskQueueId owner,
                                  TaskQueueId subsumed) const {
   std::lock_guard guard(queue_mutex_);
-  return subsumed == queue_entries_.at(owner)->owner_of || owner == subsumed;
+  return subsumed == queue_entries_.at(owner)->owner_of;
```

Comment:
nit: Would you mind adding a TODO, and filing an issue about reverting this logic once Android is fixed?

Reply:
We actually might not need to revert this as I think this make sense. waiting for @iskakaushik to confirm.

Reply:
Yup, this is good.